### PR TITLE
Add GitHub to whitelist for URL unfurling #11426

### LINKED ISSUE #11426: Add GitHub to whitelist for URL unfurling
As a user
I would like links from GitHub to be unfurled
So I can see a preview of the link posted

Description
GitHub should be added to the whitelist of domains for link unfurling. By default it should be disabled.

### PR DESCRIPTION
**Pull request by qfrank**
fixes [#11426](https://github.com/status-im/status-react/issues/11426)

### CODE REVIEW

File: protocol/urls/urls_test.go [hunk 1]
```diff
@@ -1,9 +1,8 @@
 package urls
 
 import (
-	"testing"
-
 	"github.com/stretchr/testify/require"
+	"testing"
```
Comment (cammellos) [inline protocol/urls/urls_test.go]: the linter is going to be unhappy about this one :)

Comment (qfrank) [reply protocol/urls/urls_test.go]: solved by using `goimports -w -local 'github.com/ethereum/go-ethereum,github.com/status-im/status-go,github.com/status-im/markdown' {filename}`

**Review by cammellos — APPROVED** [review]

**Comment by cammellos** [comment]
@qfrank thanks for the submission, we have introduced a list of whitelisted URLs on the client as well, would you mind adding github to it? https://github.com/status-im/status-react/blob/33e58dbca94ec8a5ee11bdfa0021ff7a3dc93265/src/status_im/utils/config.cljs#L144
Thanks!

**Comment by qfrank** [comment]
> @qfrank thanks for the submission, we have introduced a list of whitelisted URLs on the client as well, would you mind adding github to it? https://github.com/status-im/status-react/blob/33e58dbca94ec8a5ee11bdfa0021ff7a3dc93265/src/status_im/utils/config.cljs#L144
> Thanks!

NP :)

**Comment by cammellos** [comment]
@qfrank would you mind bumping the `VERSION` file? (minor version) thanks!

**Comment by qfrank** [comment]
> @qfrank would you mind bumping the `VERSION` file? (minor version) thanks!

NP, i'll investigate :)

**Comment by qfrank** [comment]
> @qfrank would you mind bumping the `VERSION` file? (minor version) thanks!

@cammellos Done ^_^

**Pull request merged**